apisix/schema_def.lua

```diff
@@ -323,7 +323,7 @@ local upstream_schema = {
         type = {
             description = "algorithms of load balancing",
             type = "string",
-            enum = {"chash", "roundrobin", "ewma", "least_conn"}
+            enum = {"chash", "roundrobin", "ewma", "least_conn", "swrr"}
```

Comment:
I'm inclined to optimize the current roundrobin loadbalancer, not creating a new one, the `roundrobin` and `swrr` are same in essence.

Reply:
This implementation is more fair when `(max weight - second max weight) / gcd` is large. 

@redynasc 
Maybe we need to put this as part of the original rr algorithm?
We can choice this one if `(max weight - second max weight) / gcd > 5`.
Thanks!

Reply:
The magical number `5` can be configured, so if we choice 0 the `swrr` will be used instead.